src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingContext.cs

```diff
@@ -16,12 +18,18 @@ internal class FormattingContext
     {
         public Uri Uri { get; set; }
 
+        public DocumentSnapshot OriginalSnapshot { get; set; }
+
         public RazorCodeDocument CodeDocument { get; set; }
 
         public SourceText SourceText => CodeDocument?.GetSourceText();
 
         public FormattingOptions Options { get; set; }
 
+        public string NewLineString => Environment.NewLine;
```

Comment:
Why have the newline string  mentioned if  it's always  gonna  be  Environment.Newline?

Reply:
I want this to eventually come from the client.